Firestore/core/src/firebase/firestore/util/path.cc

```diff
@@ -16,10 +16,41 @@
 
 #include "Firestore/core/src/firebase/firestore/util/path.h"
 
+#include "absl/strings/ascii.h"
+#include "absl/strings/string_view.h"
+
 namespace firebase {
 namespace firestore {
 namespace util {
 
+namespace {
+
+#if defined(_WIN32)  // not yet used elsewhere
```

Comment:
Optional nit: this comment is almost guaranteed to go stale.

Reply:
Yes. I have additional commits in the series that use `npos` and remove the guard (and the comment), though see below.